tests/models/Feed_test.js

```diff
@@ -1,6 +1,73 @@
 'use strict';
 
-describe('Feed', function(){
+
+var l               = require('lodash');
+var co              = require('co');
+var chai            = require('chai');
+var chaiAsPromised  = require('chai-as-promised');
+var Sequelize       = require('sequelize');
+var Promise         = Sequelize.Promise;
+var fs              = Promise.promisifyAll(require('fs'));
+var assert          = require('chai').assert;
+// var models          = require('../models');
+
+
+chai.use(chaiAsPromised);
+
+global.l = l;
+global.co = co;
+global.chai = chai;
+global.expect = chai.expect;
+global.chaiAsPromised = chaiAsPromised;
+global.fs = fs;
+
+/**
+ * get a test database
+ */
+global.getTestDB = co.wrap(function* () {
```

Comment:
This function has been moved to `tests/index.js` so every model test can make use of it.